code/modules/surgery/organs/augments_chest.dm

```diff
@@ -171,7 +171,20 @@
 		icon_state = "imp_jetpack"
```

Comment:
toggle doesn't turn off the jetpack if allow_thrust is false like you did with the other one

Reply:
Ups, forgot about this one.

Reply:
But this be toggled off after one try to move anyway.